README.md

```diff
@@ -517,6 +518,66 @@ The SAP BTP service operator project maintainers will respond to the best of the
 
 [Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
 
+## Formats of Secret Objects
+
+### Key- Value Pairs (Default)
+The binding object includes credentials returned from the broker and service instance info presented as key-value pairs.
+```bash
+#Credentials
+uri: https://my-service.authentication.eu10.hana.ondemand.com
+username: admin
+password: ********
+
+#Service instance info
+instance_guid: <instance_guid> // The service instance ID
+instance_name: my-service-btp-name // Taken from the service instance external_name field if set. Otherwise from metadata.name
+plan: sample-plan // The service plan name                
+type: sample-service  // The service offering name
+```
+
+### Credentials as JSON Object
+To show credentials returned from the broker as a JSON object, add 'secretKey: your-secret-value' to the binding spec. 
```

Comment:
it shoud be clearer that 'your-secret-value' is a sample value.
"add 'secretKey: your-secret-value' to the binding spec" seems like straight forward instruction to follow.

"...specify your desired key name in add 'secretKey' attribute in the binding spec"

Reply:
To show credentials returned from the broker as a JSON object, specify your desired key name for 'secretKey' attribute in the binding spec. 'secretKey: your-desired-key-name'

Reply:
added

Reply:
@danielforsap need to add reason and take care for Avi's comment